docs/getting-started-guides/vagrant.md

```diff
@@ -363,6 +363,24 @@ export KUBERNETES_MASTER_MEMORY=1536
 export KUBERNETES_NODE_MEMORY=2048
 ```
 
+#### I want to set proxy settings for my Kubernetes cluster boot strapping!
+
+If you are behind a proxy, you need to install vagrant proxy plugin and set the proxy settings by
+
+```sh
+vagrant plugin install vagrant-proxyconf
+export KUBERNETES_HTTP_PROXY=http://username:password@proxyaddr:proxyport
+export KUBERNETES_HTTPS_PROXY=https://username:password@proxyaddr:proxyport
+```
+
+Optionally you can specify addresses to not proxy, for example
+
+```sh
```

Comment:
I've been using the full "shell" instead of the alias "sh" -- prefer to keep the consistency/not rely on Rouge's aliasing/have something reliable to search/replace later if need be.